docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
+| `TCP_TOTAL_CACHE_ACCESSES[n]` | Req | Number of vL1D cache accesses including hits and misses (value range for n: [0-15]) |
+| `TCP_TCP_LATENCY[n]` | Cycles | Accumulated wave access latency to vL1D over all wavefronts (value range for n: [0-15]) |
+| `TCP_TCC_READ_REQ_LATENCY[n]` | Cycles | Total vL1D to L2 request latency over all wavefronts for reads and atomics with return; (value range for n: [0-15]) |
+| `TCP_TCC_WRITE_REQ_LATENCY[n]` | Cycles | Total vL1D to L2 request latency over all wavefronts for writes and atomics without return (value range for n: [0-15]) |
+| `TCP_TCC_READ_REQ[n]` | Req | Number of read requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_WRITE_REQ[n]` | Req | Number of write requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_ATOMIC_WITH_RET_REQ[n]` | Req | Number of atomic requests to L2 cache with return (value range for n: [0-15]) |
+| `TCP_TCC_ATOMIC_WITHOUT_RET_REQ[n]` | Req | Number of atomic requests to L2 cache without return (value range for n: [0-15]) |
+| `TCP_TCC_NC_READ_REQ[n]` | Req | Number of NC read requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_UC_READ_REQ[n]` | Req | Number of UC read requests to L2 cache (value range for n: [0-15] |
+| `TCP_TCC_CC_READ_REQ[n]` | Req | Number of CC read requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_RW_READ_REQ[n]` | Req | Number of RW read requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_NC_WRITE_REQ[n]` | Req | Number of NC write requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_UC_WRITE_REQ[n]` | Req | Number of UC write requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_CC_WRITE_REQ[n]` | Req | Number of CC write requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_RW_WRITE_REQ[n]` | Req | Number of RW write requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_NC_ATOMIC_REQ[n]` | Req | Number of NC atomic requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_UC_ATOMIC_REQ[n]` | Req | Number of UC atomic requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_CC_ATOMIC_REQ[n]` | Req | Number of CC atomic requests to L2 cache (value range for n: [0-15]) |
+| `TCP_TCC_RW_ATOMIC_REQ[n]` | Req | Number of RW atomic requests to L2 cache (value range for n: [0-15]) |
 
 #### TCA
 
-| Hardware Counter | Unit   | Definition                                  |
-| :----------------| :------| ------------------------------------------: |
-| `tca_cycle`        | Cycles | TCA cycles                                  |
-| `tca_busy`         | Cycles | Number of cycles  TCA has a pending request |
+| Hardware counter | Unit | Definition |
+| :----------------| :------| :-----------------|
+| `TCA_CYCLE[n]` | Cycles | Number of TCA cycles (value range for n: [0-31]) |
+| `TCA_BUSY[n]` | Cycles | Number of cycles TCA has a pending request (value range for n: [0-31]) |
 
 ### L2 cache access
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading.